CONTRIBUTING.md

```diff
@@ -42,97 +42,95 @@ Merging Pull Requests (for project contributors with write access)
-your `.profile` file on Linux/macOS.
-
-> **Windows**
-> ```bat
-> set VCPKG_ROOT=C:\path\to\vcpkg
-> set PATH=%PATH%;%VCPKG_ROOT%
-> ```
->
-> **Linux/macOS**
-> ```sh
-> export VCPKG_ROOT=/path/to/vcpkg
-> export PATH=$PATH:$VCPKG_ROOT
-> ```
-
-Finally, install the project dependencies with vcpkg.
-
-> **Windows**
-> ```bat
-> set VCPKG_DEFAULT_TRIPLET=x64-windows-static
-> vcpkg install curl[winssl]
-> ```
->
-> **Linux/macOS**
-> ```sh
-> vcpkg install curl[ssl]
-> ```
-
-### Development headers (Linux/macOS)
-On Linux/macOS the development headers for OpenSSL 1.1 must be installed to a location where CMake can find them.
-For Ubuntu 18.04 and up, you can install them directly from the main Ubuntu repository with `apt-get`. For macOS, you
-can install them with Homebrew.
-
-> **Linux (Ubuntu 18.04 and up)**
-> ```sh
-> sudo apt-get install libssl-dev
-> ```
->
-> **macOS**
-> ```sh
-> brew install openssl@1.1
-> ```
+When using vcpkg, make sure to define the `VCPKG_ROOT` environment variable to vcpkg folder before using `CMake`.
 
 ### Building and Testing
 
 #### Building the project
-First, ensure that the `VCPKG_ROOT` and `VCPKG_DEFAULT_TRIPLET` environment variables are set, as described [above](#vcpkg). This need to be defined
-any time you want to build. Then generate the build files and build as you would any standard CMake project. From the
+First, ensure that the `VCPKG_ROOT` environment variable is set, as described [above](#vcpkg). This needs to be defined
+any time you want to build using vcpkg. Then generate the build files and build as you would with any standard CMake project. From the
 repo root, run:
 
 ```sh
 mkdir build
 cd build
-cmake -Duse_default_uuid=ON ..
+cmake ..
 cmake --build .
 ```
+
+#### CMake build options
+The following CMake options are available for adding/removing project features.
+
+<table>
+<tr>
+<td>Option</td>
+<td>Description</td>
+<td>Default Value</td>
+</tr>
+<tr>
+<td>BUILD_TESTING</td>
+<td>Generates Unit Test for compilation. CMake will automatically download and build g-test.<br>After Compiling, use `ctest` to run Unit Test.</td>
+<td>OFF</td>
+</tr>
+<tr>
+<td>BUILD_STORAGE_SAMPLES</td>
+<td>Build Azure Storage clients sample application.</td>
+<td>OFF</td>
+</tr>
+<tr>
+<td>RUN_LONG_UNIT_TESTS</td>
+<td>Enables the special unit tests which takes more than 3 minutes to run. THis tests are for some specific features like the connection pool for curl transport adapter.</td>
+<td>OFF</td>
+</tr>
+<tr>
+<td>WARNINGS_AS_ERRORS</td>
+<td>Warnings will make compiling fail</td>
+<td>ON</td>
+</tr>
+<tr>
+<td>BUILD_CURL_TRANSPORT</td>
+<td>Build the curl http transport adapter. When building on Posix systems, if no other transport adapter is built, this option will be automatically turned ON</td>
+<td>OFF</td>
+</tr>
+<tr>
+<td>BUILD_DOCUMENTATION</td>
+<td>Build Doxygen documentation</td>
+<td>OFF</td>
+</tr>
+</table>
+<br>
```

Comment:
```suggestion
```

Reply:
there were 2 <br> before.
I already removed one. Removing this one would sticl the table to the next header.